README.md

```diff
@@ -178,11 +235,11 @@ Code: 200
 }
 ```
 
-### DELETE /address/{coin}
+### DELETE /v1/address/{ledger_id}

```

Comment:
We can use the `PATCH /v1/users/{username}/ledgers/{ledger_id}` endpoint instead and make the 'address' field optional. If it isn't set on the request the server deletes the entry.

Reply:
I think this is just a more complex way of doing the same thing

Reply:
It allows us merging two endpoints into one.

Reply:
Yes but its not RESTful. Sometimes separation is simpler.

Reply:
Yeah, okay.